public/core.d.ts

```diff
@@ -6416,6 +6416,204 @@ declare namespace JMap {
        */
       function remove(listenerId: string): void
     }
+
+    /**
+     * ***JMap.Event.MouseOver***
+     * 
+     * Here you can manage all mouseover related event listeners.
+     * 
+     * List of events are located in ***[[JMap.Event.MouseOver.on]]***. 
+     */
+    namespace MouseOver {
+
+      /**
+       * ***JMap.Event.MouseOver.on***
+       * 
+       * Here you have all available user events on which you can attach a listener.
+       */
+      namespace on {
+
+        /**
+         * ***JMap.Event.MouseOver.on.beforeContentProcessed***
+         * 
+         * This event is triggered each time the map is clicked, and before the mouseover content is calculated or popup opened.
+         * 
+         * This event is a special on, as it offers 2 methods which can change the mouseover behavior:
+         *  - addFeaturesToLayerSelection : add custom features to the mouseover
+         *  - removeFeaturesFromLayerSelection: used to remove a click feature from the mouseover (will not be displayed in the mouseover)
+         * 
+         * You can test the event function addFeaturesToLayerSelection, by pasting the following code in the console (adapt for your configuration):
+         * 
+         * ```ts
+         * JMap.Event.MouseOver.on.beforeContentProcessed(
+         *   "my-listener",
+         *   params => {
+         *     console.log("Mouseover selection before", params.selection[4])
+         *     params.addFeaturesToLayerSelection(4, [{
+         *       id: 58,
+         *       properties: {JMAP_FID: 58, NOM_QR: "Bois-Francs"},
+         *       type: "Feature",
+         *       geometry: {coordinates: [],type: "Polygon"}
+         *     }])
+         *     console.log("Mouseover selection after", params.selection[4])
+         *   }
+         * )
+         * ```
+         * 
+         * This listener add in every click on the map the feature, so everywhere you click
```

Comment:
This listener _**is added to**_ every click on the map, so no matter where you click the mouseover will display **_itself_**, even on a map area having no feature.